db/migrations/20221115104182_add_index_to_service_instances_service_plan_id.rb

```diff
@@ -0,0 +1,7 @@
+Sequel.migration do
+  change do
```

Comment:
There are other migration files with an `up` and `down` step instead of a single `change`, e.g. [20220802160500_index_syslog_drain_url.rb](https://github.com/cloudfoundry/cloud_controller_ng/blob/main/db/migrations/20220802160500_index_syslog_drain_url.rb)

Shall we do the same here? Yes or no? Pros vs. cons?

Reply:
Yeah, I saw this in the [Sequel docs](https://sequel.jeremyevans.net/rdoc/files/doc/migration_rdoc.html#label-A+Basic+Migration) and noticed that both approaches are used in our migration files. 

If you use `change` then Sequel will try to implicitly compose a `down` for you. The docs say this usually works for `add_index`, among others. Most of the time we don't seem to use that for adding indexes, but there are 14 examples where it is used that way - e.g. [here](https://github.com/cloudfoundry/cloud_controller_ng/blob/main/db/migrations/20180904210247_add_index_to_deployments_on_state.rb) and [here](https://github.com/cloudfoundry/cloud_controller_ng/blob/main/db/migrations/20171123191651_add_index_to_service_binding_on_name.rb).

I guess pros are that it's slightly easier to read and write, and that we've got pretty good evidence that it's safe. Maybe the con is that we haven't used it that much _recently_ for indexes and so we're less sure of it? I dunno.

Reply:
I don't have strong views either way, but I guess I have a mild preference to stick with the slightly simpler `change`.

Reply:
Yes, let's keep the `change`. And thanks for the explanation; I was not aware of this Sequel magic.